paimon-web-server/src/main/resources/mapper/UserMapper.xml

```diff
@@ -71,25 +71,26 @@ PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
     
     <select id="listUsers" parameterType="org.apache.paimon.web.server.data.model.User" resultMap="UserWithRolesResult">
 		<include refid="selectUserVo"/>
-		where u.is_delete = '0'
-		<if test="user.id != null and user.id != 0">
-			AND u.id = #{user.id}
-		</if>
-		<if test="user.username != null and user.username != ''">
-			AND u.username like concat('%', #{user.username}, '%')
-		</if>
-		<if test="user.enabled != null and user.enabled != ''">
-			AND u.enabled = #{user.enabled}
-		</if>
-		<if test="user.mobile != null and user.mobile != ''">
-			AND u.mobile like concat('%', #{user.mobile}, '%')
-		</if>
-		<if test="user.params.beginTime != null and user.params.beginTime != ''"><!-- Start time search -->
-			AND date_format(u.create_time,'%y%m%d') &gt;= date_format(#{user.params.beginTime},'%y%m%d')
-		</if>
-		<if test="user.params.endTime != null and user.params.endTime != ''"><!-- End time search -->
-			AND date_format(u.create_time,'%y%m%d') &lt;= date_format(#{user.params.endTime},'%y%m%d')
-		</if>
+		<where>
+			<if test="user.id != null and user.id != 0">
+				AND u.id = #{user.id}
```

Comment:
When `user.id` isn't null, the where condition will be `where and u.id = #{user.id}`. Is it correct?

Reply:
if user.id isn't null, the where tag will remove the first and keyword